test/integration/tests/3850-cached-templates-network-errors/files/simple.hsfiles

```diff
@@ -0,0 +1,284 @@
+{-# START_FILE {{name}}.cabal #-}
```

Comment:
Is this file needed somewhere?

Reply:
No! I left that by accident because I needed it for the first implementation of the integration test. I'll remove it.